lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -542,6 +541,11 @@ newTransactionLayer networkId = TransactionLayer
                     constructUnsignedTx networkId payload ttl rewardAcct wdrl
                         selection delta
 
+    , estimateSignedTransactionSize = \pp tx -> do
```

Comment:
Just a thought, but perhaps `estimateSignedTxSize` would be shorter but equally readable?

Reply:
Done